src/scenes/collaboration/stores/DashboardStore/DashboardStore.ts

```diff
@@ -11,11 +22,97 @@ const DashboardStore = types.compose(
     .model('DashboardStore', {
       tileDialog: types.optional(DialogModel, {}),
       tileRemoveDialog: types.optional(DialogModel, {}),
+      tileFormStore: types.optional(TileFormStore, {}),
+      updateRequest: types.optional(RequestModel, {}),
+      request: types.optional(RequestModel, {}),
       tileList: types.optional(TileList, {}),
-      dashboard: types.optional(Dashboard, {}),
-      tileFormStore: types.optional(TileFormStore, {})
+      dashboardIsEdited: true,
+      imageUrl: types.maybe(types.string),
+      videoUrl: types.maybe(types.string)
     })
-    .actions((self) => ({}))
+    .actions((self) => ({
+      fetchDashboard: flow(function* fetchDashboard(spaceId: string) {
```

Comment:
```suggestion
      fetchDashboard: flow(function* (spaceId: string) {
```
No need for extra name